the-badass-todo-list-part-1/input.js

```diff
@@ -2,6 +2,7 @@
 
 (function() {
   function drawInput(motd) {
```

Comment:
Yes it's an serious issue for design and expressiveness. That's why there are lots of abstractions like frameworks or libraries try to provide a more elegant way to do the "templating". Unfortunately, you will find those folks invented APIs like this just don't care how painful front-end developers may feel.